deployment/ansible/roles/cac-tripplanner.app/defaults/main.yml

```diff
@@ -10,3 +10,13 @@ app_sass_version: "3.4.9"
 
 root_src_dir: "/opt/app/src"
 root_static_dir: "/srv/cac"
+
+cac_python_dependencies:
```

Comment:
Is it worth it to split this into a dependencies and a dev dependencies so that things like ipython don't get installed in production?